mirascope/core/openai/call_response.py

```diff
@@ -163,7 +163,7 @@ def tool_message_params(
                 role="tool",
                 content=output,
                 tool_call_id=tool.tool_call.id,
-                name=tool._name(),  # type: ignore
+                name=tool._name(),  # pyright: ignore [reportCallIssue]
```

Comment:
same thing here, not sure why we were ignoring this, wondering why it needs to be ignored at all

Reply:
`ChatCompletionToolMessageParam` doesn't have the `name`.
Am I misunderstanding something?
https://github.com/openai/openai-python/blob/9850c169c4126fd04dc6796e4685f1b9e4924aa4/src/openai/types/chat/chat_completion_tool_message_param.py#L13-L21

Reply:
ooooh no you are correct, I was mistaken on what was being ignored.